modules/social_features/social_user/src/Plugin/Block/AccountHeaderBlock.php

```diff
@@ -385,7 +385,7 @@ public function build() {
             'title' => $this->t("View content I'm following"),
             'label' => $this->t('Following'),
             'title_classes' => '',
-            'url' => Url::fromRoute('view.following.following'),
+            'url' => Url::fromRoute('social_follow_content.overview'),
```

Comment:
Changing a route that's used for something can severely break sites. Even though we do this in a major release, we should try to stay backwards compatible.

Reply:
@Kingdutch do you propose to use the **view.following.following** route for new page base on the controller?